nature/models.py

```diff
@@ -408,7 +408,7 @@ class Meta:
         ordering = ['id']
         db_table = 'lajirekisteri'
         verbose_name = _('species')
-        verbose_name_plural = _('species')
+        verbose_name_plural = _('species register')
```

Comment:
I just realized that the model name actually does not match the `db_table`. I think it's better that we also rename the model to match the database table name. We also need to rename the model `HabitatType` of which the database table is `ltyyppirekisteri`.

@SannaElijoki  Do you think we should rename `Species` to `SpeciesRegister` and `HabitatType` to `HabitatTypeRegister`?

Reply:
Actually it is good that in app `species register` is named `species` and  `habitat type register` as `habitat type`. Otherwise there would be silly titles (free translation) like _Select a species register to edit_, and in feature form´s species observation block there would be a list of _species registers instead_ of _species_. Now that I think, in database these names should have been without "register" in the first place, like other registers "Publication", "Transaction".. But I don´t mind if English terms are renamed to match the database table name.

Reply:
You are right. In such case, I think it's also good to leave the English name without the word `register` to match the Finnish name.